Centaurus.DAL/Models/RequestRateLimitModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.DAL.Models
+{
+    public class RequestRateLimitsModel
+    {
+        public int HourLimit { get; set; }
```

Comment:
Use `uint` instead of `int` everywhere in rate limiting counters to make the calculation logic fool-proof from the start.